samples/boards/nrf52/power_mgr/README.rst

```diff
@@ -6,29 +6,23 @@ nRF5x Power management demo
 Overview
 ********
 
-This sample demonstrates a power manager app that uses the Zephyr
-power management infrastructure to enter into Low Power state.
+This sample demonstrates a power manager app that uses the Zephyr system
+power management infrastructure to manage multiple low-power states that
+are conditional on the duration of sleep.  The power mode selection is
+done automatically by residency-based policy implemented by the Zephyr
+Power Management Subsystem.
 
-This app will cycle through the following power schemes each time the system
-enters idle state:
+After all combinations have been completed the system falls of the end
```

Comment:
After all combinations **of what** have been completed?

``the system falls off the end of main()`` is an oddly non-technical description, maybe instead:
``the main() function returns``  (also there's a misspelling of ``off``)

What does pressing button 1 do after the main() function returns?  Does it start the main() function again?  If you press button 1 while the sample is running, what happens?